docs/en_US/TrainingService/AMLMode.md

```diff
@@ -5,12 +5,22 @@ NNI supports running an experiment on [AML](https://azure.microsoft.com/en-us/se
 ## Setup environment
 Step 1. Install NNI, follow the install guide [here](../Tutorial/QuickStart.md).   
 
-Step 2. Create AML account, follow the document [here](https://docs.microsoft.com/en-us/azure/machine-learning/how-to-manage-workspace-cli).
+Step 2. Create an Azure account using this [link](https://azure.microsoft.com/en-us/free/services/machine-learning/). If you already have an Azure account, skip this step.
```

Comment:
I think it's an AzureML subscription rather than an Azure account?

Reply:
This line will guide users to create an azure account. Users need to create an AzureML subcription in their azure account, see Step 5. I wll add AzureML subcription info in step 5 in the new PR.